src/modules/physics/physics.h

```diff
@@ -212,6 +214,10 @@ void lovrCylinderShapeSetRadius(CylinderShape* cylinder, float radius);
 float lovrCylinderShapeGetLength(CylinderShape* cylinder);
 void lovrCylinderShapeSetLength(CylinderShape* cylinder, float length);
 
+MeshShape* lovrMeshShapeInit(MeshShape* mesh, int vertexCount, float vertices[], int indexCount, dTriIndex indices[]);
```

Comment:
Going to try to use a non-ODE type for the index data, if possible.  Looks like the exact type of `dTriIndex` depends on various build options.

Reply:
Is there a reason to avoid ODE types in adapter code that interfaces ODE?

My thinking was if someone were to change dTRIMESH_16BIT_INDICES or dTRIMESH_GIMPACT flags that affect dTriIndex size, it would be quite nasty to track down the issue.